modules/navier_stokes/src/auxkernels/WallFunctionWallShearStressAux.C

```diff
@@ -0,0 +1,137 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "WallFunctionWallShearStressAux.h"
+
+registerMooseObject("NavierStokesApp", WallFunctionWallShearStressAux);
+
+ADReal
+findUStar3(Real mu, Real rho, ADReal u, Real dist)
```

Comment:
these should all be const arguments

move this routine to an include/INSFVMethods file so it can be shared between all WallFunction stuff